SWIPs/swip-honey_to_money.md

```diff
@@ -9,43 +9,42 @@ Created: 31-07-2019
 ---
 
 ## Simple summary 
-Swarm needs a mechanism to allow fluctuation of the exchange rate between honey (Swarms internal accounting unit) and the currency used to pay. We propose an on-chain price oracle that returns an exchange rate + validUntil tuple.  
+Swarm needs a mechanism to allow fluctuation of the exchange rate between honey (Swarm's internal accounting unit) and the currency used to pay for transferring data. This SWIP proposes an on-chain price oracle that provides an exchange rate with an associated expiration.
 
 ## Abstract 
 The ability to update prices is required in order to:
 * Facilitate experimentation with the absolute price of honey (e.g. start at a low price and gradually increase);
 * Off-set fluctuations of the currency used to pay;
-* Lower the price of Swarm in the long-run as the costs of operating a Swarm node decreases.
+* Lower the price of Swarm in the long-run as the costs of operating a Swarm node decrease.
```

Comment:
What is "the price of Swarm", exactly?

Reply:
In practice, this would be how much you pay for using Swarm (say: upload 100gb on it). As the price of honey directly translates to the price of sending messages, the `price of Swarm` is the price of honey, expressed in the currency you are paying with.